tests/uc_oss/test_uc_oss_integration.py

```diff
@@ -0,0 +1,161 @@
+import os
+import subprocess
+import sys
+
+import pandas as pd
+import pytest
+from sklearn import datasets
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+import mlflow
+
+from tests.tracking.integration_test_utils import _await_server_up_or_die
+
+
+@pytest.mark.skipif(
+    "UC_OSS_INTEGRATION" not in os.environ,
+    reason="This test is only valid w/in the github workflow integration job",
+)
+@pytest.fixture(scope="module")
+def setup_servers():
+    with subprocess.Popen([sys.executable, "-m", "mlflow", "server", "--port", "5000"]) as proc:
```

Comment:
```suggestion
    port = get_safe_port()
    with subprocess.Popen([sys.executable, "-m", "mlflow", "server", "--port", port]) as proc:
```

5000 might be unavailable.